index.js

```diff
@@ -21,15 +21,10 @@ fontmachine.makeGlyphs = function(opts, callback) {
     var stack = [];
     fontnik.load(opts.font, function(err, faces) {
         if (err) return callback(err);
-        // temp workaround: fill this out
+
```

Comment:
[`.dfont`](https://en.wikipedia.org/wiki/Datafork_TrueType) files can contain multiple fonts, although that's a format that's being phased out. [TrueType Collection](https://en.wikipedia.org/wiki/TrueType#TrueType_Collection) sounds like another older format that may be more common with CJK fonts. I think just keeping the check and error in place is fine.